packages/wallet-ui/src/components/ui/organism/Header/SendModal/SendModal.view.tsx

```diff
@@ -94,6 +139,33 @@ export const SendModalView = ({ closeModal }: Props) => {
     );
   };
 
+  const getAddrFromStarkNameQuery = async (
+    starkName: string,
+    options?: { signal?: AbortSignal },
+  ): Promise<string> => {
+    const { signal } = options || {};
+
+    return new Promise((resolve, reject) => {
```

Comment:
the abort controller doesnt consolidate the request, but just put them in queue

it means

when i first type
"alex.stark" => call snap once
then i type in 123456789 continually 
"alex12345689.stark" => call 9 time